pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -605,12 +610,22 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
       List<ProcessingException> exceptions = new ArrayList<>();
       if (numUnavailableSegments > 0) {
         String errorMessage;
+        String routingPolicy;
+        if (offlineRoutingPolicy != null && realtimeRoutingPolicy != null) {
+          routingPolicy = String.format("%s [realtime], %s [offline]", realtimeRoutingPolicy, offlineRoutingPolicy);
+        } else if (offlineRoutingPolicy != null) {
+          routingPolicy = offlineRoutingPolicy;
```

Comment:
Why not format this to have the `[offline]` tag like line 615? It will help make sure people know that it's an offline/realtime policy and it keeps consistency for the logging.